moped-editor/src/views/projects/projectView/ProjectComponents/ProjectComponentsList.js

```diff
@@ -27,20 +29,22 @@ const ProjectComponentsList = ({
 
   const onListItemClick = (component) => {
     setIsClickedComponentRelated(false);
-    // If we are editing a component, clicking on a component should not change the clicked component
+    // Clear clickedComponent and draftEditComponent when we are not selecting for edit
     if (isExpanded(component)) {
       setClickedComponent(null);
+      editDispatch({ type: "clear_draft_component" });
     } else if (isNotCreatingOrEditing) {
       setClickedComponent(component);
+      editDispatch({ type: "set_draft_component", payload: component });
```

Comment:
We used to load the component into state for the edit process when the **Edit** button was clicked. Since that button no longer exists, we need to load it when a list item is clicked and expands.